gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sun Mar 22 12:55:51 CET 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-rc-4-all.zip
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-rc-4-bin.zip
```

Comment:
We use `gradle -b wrapper.gradle wrapper` to generate gradle wrapper, which results in `gradle-xxxx-bin.zip`, so it's better to align with the existing rule ;-)

Reply:
Sorry, feel free to remove it!